backend/tests/conftest.py

```diff
@@ -1,18 +1,33 @@
+from datetime import datetime
+
 import pytest
 from fastapi.testclient import TestClient
+from sqlalchemy.orm import scoped_session
 
-from backend.constants import API_VERSION_PREFIX
+from backend.database import Session
+from backend.database.models import Project, User
 from src.backend.entrypoint import app
 
 
 @pytest.fixture()
-def test_client():
+def mock_user_id():
+    new_user = User(created_on=datetime.utcnow(), projects=[])
+    with scoped_session(Session)() as session:
+        session.add(new_user)
+        session.flush()
+        session.refresh(new_user)
+        session.commit()
+        return new_user.id
+
+
+@pytest.fixture()
+def mock_client():
```

Comment:
I think the `mock_` prefix is superfluous here. Also, why not directly returning? The variable is not necessary.

I also think this fixture should be session-scoped (as some others ; check the [doc](https://docs.pytest.org/en/7.1.x/how-to/fixtures.html?highlight=scope#scope-sharing-fixtures-across-classes-modules-packages-or-session))

Reply:
I don't think we should use `scope="session"` here. Because that will bring the side effects to the tests (e.g. legacy cookies). Overall, it's a `TestClient` and it's pretty lightweight.

Reply:
OK